tailwind.config.js

```diff
@@ -1,17 +1,18 @@
-const { colors } = require('tailwindcss/defaultTheme');
+const { colors, fontSize } = require('./old-default-theme');
```

Comment:
What's blockers are there to using the new Tailwind theme now?

Reply:
The colors are not one to one, it's different and they recommend doing this if you are reliant on the old theme now. I think they just assumed that a lot of people had their own themes defined

Reply:
I did a global search in dbt-cloud and codex I think we're using mostly custom colors. I saw a use of the default white, but I think that's all.

To me, one of the motivations to upgrading to tailwind 2 now is to get ahead of the color palette change, that way we don't have to upgrade more later. I'm willing to potentially ship some bad colors and catch the retrospectively than blocking ourselves on [solidifying our color palette on tailwind 2's color palette](https://fishtownanalytics.slack.com/archives/C01BLFXN8QN/p1614006481003200). What are your thoughts on this?

Reply:
Are you sure? https://github.com/fishtown-analytics/dbt-cloud/blob/master/frontend/components/connection/ConnectionForm.tsx#L32
https://github.com/fishtown-analytics/dbt-cloud/blob/master/service-token/TokenSuccess.tsx#L42

Honestly it's up to the team. My understand was that we were going to get a full color palette and we could just replace `old-default-theme.js` with `actual-theme.js` and nothing would change. This isn't really upgrading anything to me since this is just what we'd be doing anyway when we get the new theme.

Reply:
Okay, we can upgrade our color palette later!